### PR TITLE
Rework governance/ update admin-module NTRN-70 sdk47

### PR DESCRIPTION
**Pull request by quasisamurai**
[TASK](https://hadronlabs.atlassian.net/browse/NTRN-70)

[Admin module PR](https://github.com/neutron-org/admin-module/pull/2)
[Neutron PR](https://github.com/neutron-org/neutron/pull/303)
[Neutron SDK PR](https://github.com/neutron-org/neutron-sdk/pull/107)
[Integration tests](https://github.com/neutron-org/neutron-integration-tests/pull/196)

### CODE REVIEW

File: src/testcases/parallel/governance.test.ts
Comment (pr0n00gler) [inline src/testcases/parallel/governance.test.ts:1]: Please add a tests to check non-whitelisted message will not be executed in proposal

**Review by pr0n00gler — CHANGES_REQUESTED** [review]
Also attach a successful testrun after the fix

File: src/testcases/parallel/governance.test.ts
Comment (sotnikov-s) [reply src/testcases/parallel/governance.test.ts:1]: is it possible to add more tests around authority/signers logic?

**Review by sotnikov-s — APPROVED** [review]

**Comment by quasisamurai** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/6097481262/job/16550121022

**Pull request merged**